index.js

```diff
@@ -183,6 +184,41 @@ class GithubScm extends Scm {
         };
     }
 
+    /**
+     * Promise to wait a certain number of seconds
+     *
+     * Might make this centralized for other tests to leverage
+     *
+     * @method promiseToWait
+     * @param  {Number}      timeToWait  Number of seconds to wait before continuing the chain
+     * @return {Promise}
+     */
+    promiseToWait(timeToWait) {
+        return new Promise((resolve) => {
+            setTimeout(() => resolve(), timeToWait * 1000);
+        });
+    }
+
+    /**
+     * Get PR mergeability
+     * @async  getPrMergeable
+     * @param  {Object}   config
+     * @param  {String}   config.scmUri     The scmUri to get PR info of
+     * @param  {String}   config.token      The token used to authenticate to the SCM
+     * @param  {Integer}  config.prNum      The PR number used to fetch the PR
+     * @return {Promise}
+     */
+    async getPrMergeable({ scmUri, token, prNum }) {
+        const { mergeable } = await this.getPrInfo({ scmUri, token, prNum });
+
+        if (mergeable !== null) {
+            return mergeable;
+        }
+        await this.promiseToWait(POLLING_INTERVAL);
```

Comment:
Do we need any max attempt of the polling?

Reply:
Thanks. I fixed it.